src/ipbb/depparser/_fileparser.py

```diff
@@ -463,13 +463,27 @@ def _parse_file(self, aPackage, aComponent, aDepFileName, aParentDep):
         self._depregistry[lDepFilePath] = lCurrentFile
 
         with open(lDepFilePath) as lDepFile:
-            for lLineNr, lLine in enumerate(lDepFile):
+            lLinePrev = ""
+            for lLineNr, lLineRaw in enumerate(lDepFile):
 
                 lDepInfo = (lCurrentFile.full_path(), lLineNr)
 
                 # --------------------------------------------------------------
                 # Pre-processing
                 try:
+                    # See if this is a split line or not.
+                    LINE_SPLIT_CHARACTER = "\\"
+                    if lLineRaw.strip().endswith(LINE_SPLIT_CHARACTER):
+                        lLinePrev += lLineRaw.strip()[:-1]
+                        continue
+                    else:
+                        lLine = (lLinePrev.strip() + " " + lLineRaw.strip()).strip()
+                        if lLinePrev:
```

Comment:
Thanks for this change! Can you drop these debug lines?

Reply:
Done. Sorry about that.